.github/workflows/release.yml

```diff
@@ -0,0 +1,49 @@
+name: Release
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+concurrency:
+  group: build-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04] # TODO: macos-11
```

Comment:
we probably need an ubuntu-18.04 arm version too.

Reply:
Added a TODO. Will require an ARM runner.

Reply:
or can be done with docker build + qemu